src/networktables/NetworkTable.cpp

```diff
@@ -43,6 +42,8 @@ void NetworkTable::SetTeam(int team) {
   char tmp[30];
 #ifdef _MSC_VER
   sprintf_s(tmp, "roboRIO-%d-FRC.local\n", team);
+#elif defined(__ANDROID__)
+  snprintf(tmp, 30, "roboRIO-%d-FRC.local\n",team);
```

Comment:
I'll try that. I tried including ```cstdio``` and ```stdio``` without those working, but I'll try ```cstring``` when I get home. Is it save to include ```cstring``` for everyone, or does that have to be in its own ```#ifdef```?